src/spyglass/spikesorting/v1/metric_curation.py

```diff
@@ -259,37 +260,40 @@ def make(self, key):
 
         AnalysisNwbfile()._creation_times["pre_create_time"] = time()
         # FETCH
-        nwb_file_name = (
-            SpikeSortingSelection * MetricCurationSelection & key
-        ).fetch1("nwb_file_name")
-
-        # TODO: reduce fetch calls on same tables
-        waveform_params = (
-            WaveformParameters * MetricCurationSelection & key
-        ).fetch1("waveform_params")
-        metric_params = (
-            MetricParameters * MetricCurationSelection & key
-        ).fetch1("metric_params")
-        label_params, merge_params = (
-            MetricCurationParameters * MetricCurationSelection & key
-        ).fetch1("label_params", "merge_params")
-        sorting_id, curation_id = (MetricCurationSelection & key).fetch1(
-            "sorting_id", "curation_id"
+        query = (
```

Comment:
Have you tested this query?

Reply:
Yes - I ran to make sure they were join compatible and contained all relevant fields